addons/mail/static/src/components/chatter_topbar/chatter_topbar.scss

```diff
@@ -67,14 +67,17 @@
 .o_ChatterTopbar_button {
     border-radius: 0;
 
-    &:hover {
-        background-color: gray('300');
+    @include hover-focus() {
```

Comment:
@seb-odoo ,  I'd prefer:
```scss
&:hover:not(.active), &:focus:not(:hover)  {
        background-color: gray('200');
    }
}
```
Final result should:
- highlight any buttons on keyboard navigation
- highlight "not active" btns on mouse-hover only

IMO there's no point to highlight **active** buttons on mouse-hover  too.
The behaviour is clear, changing active bg on :hover just add another level of complexity

Reply:
Done, thanks for checking it